test/testdrive/storage-statistics.td

```diff
@@ -99,26 +99,44 @@ users          true      false   true  true  false
 
 # Test upsert
 
-# Note that we see 9 messages received, but only 3 updates, because there are only
-# 3 active keys, as all the messages are received in 1 second.
-> SELECT s.name, MIN(u.snapshot_committed), SUM(u.messages_received), SUM(u.updates_staged), SUM(u.updates_committed), SUM(u.bytes_received) > 0
+# Note that we always count 9 messages received, but can see as low as 3 updates.
+# This is because there are 3 active keys, as all the messages could be received in 1 second.
+# There could also be up to 11 updates, as some messages cause inserts and deletes.
```

Comment:
someone is gonna have to check my math here, the messages are on line 30

Reply:
I think your `11` for the maximum number of "raw" update is correct, we have `5 + 1 + 1 + 2 + 2`, five initial inserts, for the five different keys, two messages that cause only a retraction, and then two updates that cause both a retraction and an update.

Reply:
Also checked that `BETWEEN` is inclusive for both the lower and upper... because I don't know SQL 🙈

Reply:
it IS inclusive lol, i had to check too! I will copy your comment as a explanation of the number 11!2